privacy-dashboard/privacy-dashboard-impl/src/main/java/com/duckduckgo/privacy/dashboard/impl/animations/BrowserTrackersAnimatorHelper.kt

```diff
@@ -1,257 +0,0 @@
-/*
- * Copyright (c) 2022 DuckDuckGo
- *
- * Licensed under the Apache License, Version 2.0 (the "License");
- * you may not use this file except in compliance with the License.
- * You may obtain a copy of the License at
- *
- *     http://www.apache.org/licenses/LICENSE-2.0
- *
- * Unless required by applicable law or agreed to in writing, software
- * distributed under the License is distributed on an "AS IS" BASIS,
- * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
- * See the License for the specific language governing permissions and
- * limitations under the License.
- */
-
-package com.duckduckgo.privacy.dashboard.impl.animations
-
-import android.animation.Animator
-import android.animation.Animator.AnimatorListener
-import android.animation.AnimatorSet
-import android.animation.ObjectAnimator
-import android.content.Context
-import android.view.View
-import com.airbnb.lottie.LottieAnimationView
-import com.duckduckgo.app.trackerdetection.model.Entity
-import com.duckduckgo.di.scopes.FragmentScope
-import com.duckduckgo.mobile.android.ui.store.AppTheme
-import com.duckduckgo.privacy.dashboard.api.animations.BrowserTrackersAnimatorHelper
-import com.duckduckgo.privacy.dashboard.api.animations.TrackersAnimatorListener
-import com.duckduckgo.privacy.dashboard.impl.R
-import com.duckduckgo.privacy.dashboard.impl.animations.TrackerLogo.ImageLogo
-import com.duckduckgo.privacy.dashboard.impl.animations.TrackerLogo.LetterLogo
-import com.duckduckgo.privacy.dashboard.impl.animations.TrackerLogo.StackedLogo
-import com.squareup.anvil.annotations.ContributesBinding
-import javax.inject.Inject
-
-@ContributesBinding(FragmentScope::class)
-class BrowserLottieTrackersAnimatorHelper @Inject constructor(
```

Comment:
Not sure why this is detected as class removed. This has been moved back into `:app` module as commnted over zoom. It doesn't make sense to have this logic as part of the privacy dashboard since it's only related to the omnibar animations.